### ISSUE TITLE
Adding aduh95 to linters team

### ISSUE THREAD

**Issue by nschonni** [nodejs/remark-preset-lint-node#160]
@Trott I don't think we have a particular repo for these issues, but since @aduh95 seems to have some interested and experience, it think it makes sense to add them.

**Comment by Trott** [comment]
I agree (especially since he has indicated his own agreement with the 👍 ). I'll move this to the admin repo for visibility.
